src/test/subscription/t/030_rep_ddls.pl

```diff
@@ -13,6 +13,11 @@
 $node_publisher->append_conf('postgresql.conf', 'autovacuum = off');
 $node_publisher->start;
 
+my $node_publisher2 = PostgreSQL::Test::Cluster->new('publisher2');
```

Comment:
There is no need to start up two more servers to run the new test case. You can set the test up in a new database instead.

$node_publisher->safe_psql('postgres', "CREATE DATABASE db1;");
$node_publisher->safe_psql('db1', "CREATE PUBLICATION ...");
...